### ISSUE TITLE
Apps too slow in non built-in laptop monitor

### ISSUE THREAD

**Issue by RaunySouza** [pop-os/pop#2441]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

**Hardware**
Laptop Dell G15 5515 - Ryzen 7 5800H, Nvidia RTX3600M (Driver 510.68.02), 16GB.
Setup 3 monitors: Built-in laptop monitor, Gygabyte G24F (Displayport), Dell P2219H (HDMI).
 
**Issue/Bug Description:**
Some apps (Microsoft Edge, Slack, Gnome Extensions) are running too slow on non built-in monitor when primary display is NOT the built-in one.

I noticed this problem after some update a few days ago. Before version 22.04 I've never had this problems and even after upgrade to latest release everything was working perfectly, but one or two weeks ago this problem showed up.

**Steps to reproduce (if you know):**
Make any non-built-in monitor as your primary display and then open (or drag) some app in those monitors.
The apps get unusable. Only after dragging them to built-in monitor they get normal.

**Expected behavior:**
App run smoothly in any monitor.

**Comment by leviport** [comment]
This looks like a duplicate of https://github.com/pop-os/nvidia-graphics-drivers/issues/61

Does everything run more smoothly if you switch to Nvidia graphics mode?

**Comment by RaunySouza** [comment]
Well, the symptoms are slight different from what was reported in this other thread, but indeed, disabling hybrid graphics mode worked here. This is a bit weird considering that I didn't notice any problem even using driver 510 and latest LTS until days ago.

So, the problem was partially solved, at least I can perfectly use the laptop now.

Thank you.
